src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -882,7 +882,21 @@ static void Goo()
                     {
                     }
                 }
-                """, "null", "true", "false", "await");
+                """;
+
+            foreach (var keywordKind in SyntaxFacts.GetKeywordKinds())
+            {
+                var keywordText = SyntaxFacts.GetText(keywordKind);
```

Comment:
This is reusing the same logic as the actual product code, which isn't great for a test at all, but I cannot think of a better solution here. Another option is to manually list all keywords we need to check for, but this is an unmanageble nightmare. We sort of want keyword list to be tight to `SyntaxFacts.GetKeywordKinds`, so as new keywords are added to the language we immediately test that they are in the list. Same thing on VB side